python/py-cucumber-tag-expressions/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        cucumber tag-expressions-python ace002b
+version             2.0.2
+revision            0
+name                py-cucumber-tag-expressions
+categories-append   devel
+
+license             MIT
+maintainers         nomaintainer
+platforms           darwin
+
+description         Tag Expression Parser for Python
+
+long_description    Cucumber can be used to implement automated tests\
+                    based on scenarios described in your Gherkin\
+                    feature files.
+
+checksums           rmd160  77f0b50aff47cab4994deb94d8f3c9ac00d05a67 \
+                    sha256  35fb5e9445a2913b7a02a7ab691dd0acdf3d5664118717d2b9c1becec9497eea \
+                    size    32498
+
+python.versions     37 38
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools_scm
+
+    depends_run-append \
+                    port:py${python.version}-coverage \
+                    port:py${python.version}-invoke \
+                    port:py${python.version}-path \
+                    port:py${python.version}-pylint \
+                    port:py${python.version}-tox
+
+    depends_test-append \
+                    port:py${python.version}-pytest \
+                    port:py${python.version}-pytest-html
+
+    post-destroot {
+        set docdir ${prefix}/share/doc/${subport}
+        xinstall -d ${destroot}${docdir}
+        xinstall -m 0644 -W ${worksrcpath} README.rst LICENSE \
+            ${destroot}${docdir}
+    }
+
+    test.run        yes
+    test.env        PYTHONPATH=${worksrcpath}/build/lib
```

Comment:
this doesn't run the tests as far as I can tell (```error: invalid command 'pytest'```); please double check

Reply:
Adding your code from above works for this one. Thanks.